### PR TITLE
Improve rollover logging for error scenarios

### PR DESCRIPTION
**Pull request by danhermann**

### CODE REVIEW

File: server/src/main/java/org/elasticsearch/action/admin/indices/rollover/MetadataRolloverService.java [hunk 1]
```diff
@@ -84,6 +84,11 @@ private RolloverResult(String rolloverIndexName, String sourceIndexName, Cluster
             this.sourceIndexName = sourceIndexName;
             this.clusterState = clusterState;
         }
+
+        @Override
+        public String toString() {
+            return String.format(Locale.ROOT, "rollover index name [%s], source index name [%s]", rolloverIndexName, sourceIndexName);
```
Comment (dakrone) [inline server/src/main/java/org/elasticsearch/action/admin/indices/rollover/MetadataRolloverService.java:90]: Can you add the cluster stat version to this, perhaps:

```suggestion
            return String.format(Locale.ROOT, "[%d] rollover index name [%s], source index name [%s]", clusterState.version(), rolloverIndexName, sourceIndexName);
```

(or some other format)

**Review by dakrone — APPROVED** [review]
LGTM, I left one suggestion, thanks Dan!

**Comment by danhermann** [comment]
@elasticmachine update branch

**Comment by danhermann** [comment]
Thanks, @dakrone!

**Pull request merged**